examples/experimental/trial/gbt_estimator/model_def.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+
+from IPython.display import clear_output
```

Comment:
is this used?

same with matplotlib

Reply:
good catch @katport; I removed the unnecessary packages